examples/tutorials/vectors.py

```diff
@@ -0,0 +1,461 @@
+# list will increase accordingly.
+#
+# Additionally, we change the style of the vector to include a red
+# arrow head and increase the thickness of the line. A list of different
+# styling attributes can be found in
+# :doc:`Vector heads and tails </gallery/lines/vector_heads_tails>`.
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X10c/10c",
+    frame="ag",
+    x=[2, 4],
+    y=[8, 1],
+    direction=[[-45, 23], [6, 3]],
+    style="v0.6c+e",
+    pen="2p",
+    color="red3",
+)
+fig.show()
+
+########################################################################################
+# The default unit of vector length is centimeters,
+# however, this can be changed to inches or points. Note that, in PyGMT,
+# one point is defined as 1/72 inch.
+#
+# In this example, the graphed region is 5in X 5in, but
+# the length of the first vector is still graphed in centimeters.
+# Using ``pygmt.config(PROJ_LENGTH_UNIT="i")``, the default unit
+# can be changed to inches in the second plotted vector.
+
+fig = pygmt.Figure()
+# Vector 1 with default unit as cm
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X5i/5i",
+    frame="a",
+    x=2,
+    y=8,
+    direction=[[0], [3]],
+    style="v1c+e",
+    pen="2p",
+    color="red3",
+)
+# Vector 2 after changing default unit to in
+with pygmt.config(PROJ_LENGTH_UNIT="i"):
+    fig.plot(
+        x=2,
+        y=7,
+        direction=[[0], [3]],
+        style="v1c+e",
+        pen="2p",
+        color="red3",
+    )
+fig.show()
+
+########################################################################################
+# Vectors can also be plotted by including all the information
+# about a vector in a single list. However, this requires creating
+# a list for all vectors and passing it into a numpy array object.
+# Each vector list contains the information structured as:
+# ``[x_start, y_start, direction_degrees, length]``
+#
+# If this approach is chosen, the ``data`` parameter must be
+# used instead of ``x``, ``y`` and  ``direction``.
+
+# Create a list of lists that include each vector information
+vectors = np.array([[2, 3, 45, 4]])  # vectors structure: [[ 2  3 45  4]]
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X10c/10c",
+    frame="ag",
+    data=vectors,
+    style="v0.6c+e",
+    pen="2p",
+    color="red3",
+)
+fig.show()
+
+########################################################################################
+# Using the functionality mentioned in the previous example,
+# multiple vectors can be plotted at the same time. Another
+# vector could be simply added to the 2d numpy array object
+# and passed using ``data`` parameter.
+
+# vector specifications structured as: [x_start, y_start, direction_degrees, length]
+vector_1 = [2, 3, 45, 4]
+vector_2 = [7.5, 8.3, -120.5, 7.2]
+# Create a list of lists that include each vector information
+vectors = np.array([vector_1] + [vector_2])
```

Comment:
Both work, but I feel the following one is more readable.
```suggestion
vectors = np.array([vector_1, vector_2])
```